terraform/accounts/main/alarms.tf

```diff
@@ -22,9 +22,9 @@ resource "aws_cloudwatch_metric_alarm" "jenkins_data_volume_disk_space" {
     path = "/data"
   }
 
-  // For for every 60 seconds
+  // For for every 300 seconds
```

Comment:
I know these were already there, but can we get rid of the extra `for` in the comments 😸

Reply:
I did think about this, actually. And wondered if it was meant as `for "for every 300 seconds"` 🤣 I will clean up